cli.go

```diff
@@ -40,6 +40,12 @@ func NewCLI() *cli.App {
 				Aliases: []string{"R"},
 				EnvVars: []string{"CYCLIST_REDIS_URL", "REDIS_URL"},
 			},
+			&cli.UintFlag{
```

Comment:
I ended up going with uint because that's what is used at the redis level.  I agreed that a duration flag would be a better fit for how I think of it.  +0 ?  I'll gladly switch it to a duration flag if yinz feel strongly about it :heart: